rfcs/text/0033/process.yml

```diff
@@ -51,6 +51,44 @@
 
         Best efforts are made to ensure complete lines are captured in these events. Assumptions should NOT be made that multiple lines will appear in the same event. TTY output may contain terminal control codes such as for cursor movement, so some string queries may not match due to terminal codes inserted between characters of a word.
 
+    - name: io.total_bytes_captured
+      level: extended
+      type: number
+      description: >
+        The total number of bytes captured in this event.
+
+    - name: io.total_bytes_skipped
+      level: extended
+      type: number
+      description: >
+        The total number of bytes skipped in this event.
+
+    - name: io.max_kilobytes_per_process_exceeded
+      level: extended
+      type: boolean
+      description: >
+        If true, the process producing the output has exceeded the max_kilobytes_per_process configuration setting.
+
+    - name: io.bytes_skipped
+      level: extended
+      type: object
+      description: >
+        An array of byte offsets and lengths denoting where IO data may have been skipped.
```

Comment:
I think this is clearer (no maybe about it):
"An array of byte offsets and lengths denoting where IO data has been skipped."